app/src/components/account/UserMenu.vue

```diff
@@ -37,7 +37,11 @@
         </div>
         <div
           v-for="address in addresses"
-          :key="address.address.concat(`-${address.networkId}`)"
+          :key="
+            address.address.concat(
```

Comment:
why can't you just use the address?

Reply:
because you can have the same address from different session types and then it will throw a duplicated keys error

Reply:
but yeah, networkId is definitely extra... although Polkadot addresses are kind of similar to Bitcoins... thinking ahead here

Reply:
This refactor could go into another PR... can break many things actually